src/settings.cpp

```diff
@@ -859,14 +870,22 @@ if ( GetFlagIniSet ( IniXMLDocument, "server", "defcentservaddr", bValue ) )
     }
 
     // welcome message (command line overwrites setting file)
-    if ( pServer->GetWelcomeMessage().isEmpty() )
+    //if ( pServer->GetWelcomeMessage().isEmpty() )
+    if ( !CommandLineOptions.contains( "--welcomemessage" ) )
     {
         pServer->SetWelcomeMessage ( FromBase64ToString ( GetIniSetting ( IniXMLDocument, "server", "welcome" ) ) );
     }
 
     // window position of the main window
     vecWindowPosMain = FromBase64ToByteArray (
         GetIniSetting ( IniXMLDocument, "server", "winposmain_base64" ) );
+
+    // base recording directory (command line overwrites setting file)
```

Comment:
Since now all settings are overwritten by the command line, maybe we should remove all the "_(command line overwrites setting file)_", what do you think?

Reply:
It could go as a comment on the call in `main.cpp` - I think it needs mentioning.

Reply:
Ok, let's do that later on. I'll now merge your code.

Reply:
I just saw that you already put it in main.cpp -> done.